src/main/java/com/epam/reportportal/aspect/StepRequestUtils.java

```diff
@@ -62,4 +66,12 @@ static FinishTestItemRQ buildFinishStepRequest(String status, Date endTime) {
 		rq.setStatus(status);
 		return rq;
 	}
+
+	private static Set<ItemAttributesRQ> createStepAttributes(MethodSignature methodSignature) {
+		Attributes attributesAnnotation = methodSignature.getMethod().getAnnotation(Attributes.class);
```

Comment:
Why do you get only `Attributes` annotation what if I want to add a single attribute? Your `Attribute` annotation allows that (setting on method by its own).

Reply:
Annotation targets updated. Now only @Attributes annotation can be attached to the methods the remaining annotations can be only used inside this annotation